tests/end2end/test_invocations.py

```diff
@@ -417,3 +417,23 @@ def test_referrer(quteproc_new, server, server2, request, value, expected):
             expected = expected.replace(key, str(val))
 
     assert headers.get('Referer') == expected
+
+
+@pytest.mark.qtwebkit_skip
+@utils.qt514
+def test_preferred_colorscheme(request, quteproc_new):
+    """Make sure the the preferred colorscheme is set."""
+    args = _base_args(request.config) + [
+        '--temp-basedir',
+        '-s', 'colors.webpage.prefers_color_scheme_dark', 'true',
+    ]
+    quteproc_new.start(args)
+
+    quteproc_new.open_path("qute://version")
+    quteproc_new.send_cmd(':jseval --world main '
+                          'matchMedia("(prefers-color-scheme: dark)").matches')
+    line = quteproc_new.wait_for(message='True')
+    line.expected = True
+
+    quteproc_new.send_cmd(':quit')
+    quteproc_new.wait_for_quit()
```

Comment:
The `quteproc_new` [fixture](https://docs.pytest.org/en/stable/fixture.html) already takes care of cleaning up the qutebrowser process - I think (some of) the other tests doing this manually is an accident.